portsyncd/linksync.cpp

```diff
@@ -246,15 +248,16 @@ void LinkSync::onMsg(int nlmsg_type, struct nl_object *obj)
     /* front panel interfaces: Check if the port is in the PORT_TABLE
      * non-front panel interfaces such as eth0, lo which are not in the
      * PORT_TABLE are ignored. */
-    vector<FieldValueTuple> temp;
     if (m_portTable.get(key, temp))
     {
         g_portSet.erase(key);
         FieldValueTuple tuple("state", "ok");
         vector<FieldValueTuple> vector;
         vector.push_back(tuple);
+        FieldValueTuple op("oper_status", oper ? "up" : "down");
```

Comment:
Rename "oper_status" attribute to "netdev_oper_status" to avoid confusion with "oper_status" in "PORT_TABLE" in app db.

Reply:
Fixed